core/src/main/java/it/pagopa/selfcare/mscore/core/strategy/CreateInstitutionStrategyIpa.java

```diff
@@ -117,13 +122,15 @@ private Institution mappingToInstitutionIPAAoo(CreateInstitutionStrategyInput st
         newInstitution.setSubunitType(InstitutionPaSubunitType.AOO.name());
         newInstitution.setParentDescription(institutionProxyInfo.getDescription());
         newInstitution.setRootParentId(rootParentInstitutionId);
-
         newInstitution.setExternalId(getExternalId(strategyInput));
         newInstitution.setOrigin(Optional.ofNullable(areaOrganizzativaOmogenea.getOrigin())
                 .map(Origin::name)
                 .orElse(null));
         newInstitution.setCreatedAt(OffsetDateTime.now());
-
+        newInstitution.setCity(geotax.getDescription().replaceAll(DESCRIPTION_TO_REPLACE_REGEX, ""));
```

Comment:
you could check if description if not null before replacing with text for example using Optional.ofNullable...it is a potential nullpointerexception